osu.Game.Rulesets.Osu/Objects/Drawables/DrawableOsuHitObject.cs

```diff
@@ -51,6 +52,19 @@ protected virtual void UpdatePreemptState()
         protected virtual void UpdateCurrentState(ArmedState state)
         {
         }
+
+        private readonly WeakReference<OsuInputManager> osuActionInputManager = new WeakReference<OsuInputManager>(null);

```

Comment:
Actually, since it's coming from the parent, I guess it can't really ever be released without the child also being released.  I'll just change it to a lazy property.